WordPress/twentytwentyfour
# Sidebar: cleanup and enhancements

**Description**



This PR removes the sidebar-writer pattern because it's virtually the same as this one but with french text. I changed a few blocks:

- The image is now the avatar
- The post meta uses the block pattern
- The useful links use the nav block instead of another query loop

**Screenshots**



Before:

<img width="1317" alt="Screenshot 2023-09-15 at 17 13 51" src="https://github.com/WordPress/twentytwentyfour/assets/3593343/46c750a9-8d52-47c4-abc1-3b537e6f27a1">

After:

<img width="1406" alt="Screenshot 2023-09-15 at 17 13 09" src="https://github.com/WordPress/twentytwentyfour/assets/3593343/05156315-f72b-4251-ba42-b91be0ec15ab">

**Testing Instructions**






Create a post and select single with sidebar as the template. Make sure to add a description to the author page in your wp install to see all the content
